.prettierignore

```diff
@@ -1 +1,2 @@
-/node_modules/
\ No newline at end of file
+/node_modules/
+*.md
```

Comment:
Prettier is needed to maintain the md files in good order. We should not put them in the ignore file. If there is a certain prettier rule that is causing a problem we should look into that rule and see if we need to change it